po/pt.po

```diff
@@ -9,19 +9,19 @@ msgstr ""
 "Project-Id-Version: \n"
 "Report-Msgid-Bugs-To: \n"
 "POT-Creation-Date: 2019-10-17 21:21+0200\n"
-"PO-Revision-Date: 2019-03-31 10:19+0200\n"
-"Last-Translator: Raí B. Toffoletto <rai@raibtoffoletto.com>\n"
+"PO-Revision-Date: 2019-11-03 21:59+0000\n"
+"Last-Translator: Hugo Carvalho <hugokarvalho@hotmail.com>\n"
 "Language-Team: \n"
-"Language: pt\n"
+"Language: pt_PT\n"
```

Comment:
```suggestion
"Language: pt\n"
```

I guess lefting `pt` is fine, for example: https://github.com/elementary/sideload/blob/7fecf730d716f0fab41c5ddf35d317280a017c74/po/extra/pt.po#L15

Reply:
My mistake editing the po file in Poedit.
Done. Thanks